### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by srz-zumix** [google/googletest#997]
Compile error, Commit d62d6c6
https://travis-ci.org/google/googletest/builds/197117776

```
In file included from /home/travis/build/google/googletest/googletest/include/gtest/gtest.h:66:0,
                 from /home/travis/build/google/googletest/googletest/test/gtest-typed-test_test.h:35,
                 from /home/travis/build/google/googletest/googletest/test/gtest-typed-test_test.cc:32:
/home/travis/build/google/googletest/googletest/include/gtest/gtest-typed-test.h:245:7: error: expected initializer before ‘GTEST_ATTRIBUTE_UNUSED’
       GTEST_ATTRIBUTE_UNUSED = \
```

**Comment by srz-zumix** [comment]
Should be fixed by PR #996

**Comment by hethi** [comment]
As the PR is merged, you may close this issue?
